src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -92,60 +98,94 @@ const PIACollectionUseAndDisclosure = () => {
           onChangeHandler={(updatedData) => {
             stateChangeHandler(updatedData, 'steps');
           }}
+          readOnly={isReadOnly}
         />
       </section>
 
       <h3 className="pt-5 pb-2">{Messages.CollectionNotice.Title.en}</h3>
 
       <section className=" card pt-5 px-5">
         <div className="form-group">
-          <label htmlFor="collectionNoticeDrafter">
-            {Messages.CollectionNotice.DrafterInput.Title.PartOne.en}
-            <a
-              href={Messages.CollectionNotice.DrafterInput.Link.en}
-              rel="noreferrer external"
-              target="_blank"
-            >
+          {!isReadOnly ? (
+            <label htmlFor="collectionNoticeDrafter">
+              {Messages.CollectionNotice.DrafterInput.Title.PartOne.en}
```

Comment:
From Figma, this title seems to be part of view as well? Correct me if I am wrong.

Reply:
It is still there, it just becomes an `h4` instead of being inside of a `label`